src/corelibs/U2Lang/src/model/DbiDataHandler.h

```diff
@@ -37,18 +37,21 @@ class U2LANG_EXPORT DbiDataHandler : public QSharedData {
     friend class DbiDataStorage;
 
 public:
-    DbiDataHandler(const U2EntityRef &entRef, U2ObjectDbi *dbi, bool useGC);
+    DbiDataHandler(const U2EntityRef &entityRef, U2ObjectDbi *dbi, bool useGC);
     virtual ~DbiDataHandler();
 
     int getReferenceCount() const;
     U2DbiRef getDbiRef() const;
 
+    /** Returns entity-ref this data handler holds. */
+    U2EntityRef getEntityRef() const;
```

Comment:
I'd rather make the return value `const U2EntityRef&` to avoid copying

Reply:
Good, fixed